src/ui/deck.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 The boardgame.io Authors
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import "./deck.css";
+
+const Deck = ({ cards, ...rest }) => (
+  <div className="bgio-deck" {...rest}>
+    {cards.map((card, i) =>
+      React.cloneElement(card, {
+        key: i,
+        className: i > 0 ? "no-hover" : "",
+        isFaceUp: i === 0, // Only the top card should ever be face up
+        style: {
+          position: i ? "absolute" : "inherit",
+          left: i * 3,
```

Comment:
Let's add a `splayWidth` prop to `Deck` to allow the user to adjust this magic number `3` here? That will also allow the user to disable splay altogether by setting it to `0` if they desire viewing just the top card and nothing else.